build.sbt

```diff
@@ -894,6 +961,7 @@ def http4sCrossProject(name: String, crossType: CrossType) =
     )
     .enablePlugins(Http4sPlugin)
     .jsConfigure(_.disablePlugins(DoctestPlugin))
+    .transform(_.dependsOn(scalafixInternalRules % ScalafixConfig))
```

Comment:
```suggestion
    .dependsOn(scalafixInternalRules % ScalafixConfig)
```

Reply:
You can't use `dependsOn` directly there because it's a cross project - it needs `configure`.

Reply:
`dependsOn` works for cross projects, see any of the cross projects above. But, maybe not with scalafix :)

Reply:
Only when depending on another cross-project I think - here it's depending on a plain jvm-only `Project`(which is allowed with scalafix)